src/action-creators/splitView.js

```diff
@@ -0,0 +1,6 @@
+export const updateSplitPosition = splitPosition => dispatch => {
```

Comment:
Please match the filename with the action-creator name.

Reply:
Got it. I'll update it